config/modules/definitelyTypedTransformer/webpack.functions.js

```diff
@@ -1,10 +1,12 @@
 const path = require("path");
 const { merge } = require("webpack-merge");
 const base = require("../base/webpack.base");
+const DetermineDevToolFromEnvironmentDebugMode = require("../../utils/devtool");
 
 module.exports = merge(base({
     tsConfigFile: 'config/modules/definitelyTypedTransformer/tsconfig.json'
 }), {
+    devtool: DetermineDevToolFromEnvironmentDebugMode(),
```

Comment:
We could move this one in the base webpack.

Reply:
I agree, I'll do it

Reply:
done